requirements/prod-requirements.in

```diff
@@ -9,7 +9,7 @@ errand-boy==0.3.8
 setproctitle==1.1.9
 uWSGI==2.0.11.2
 ipython==5.2.2
-ndg-httpsclient==0.5.0
+ndg-httpsclient==0.5.1
```

Comment:
This will come up in the discussion tomorrow, but I'd suggest simply _not pinning_, any time we make changes to the `*.in` files. If there does happen to be an upper or lower version limit we know doesn't work, we can use `<`/`>`.  Then when we run `make requirements` we have a chance to review any actual version updates before committing.

Reply:
Oh, I see you mentioned this in the PR comment. I can offline with you.